mix.exs

```diff
@@ -1,46 +1,52 @@
 defmodule ExPhoneNumber.Mixfile do
   use Mix.Project
 
-  @version "0.1.1"
+  @version "0.2.0"
 
   def project do
-    [app: :ex_phone_number,
-     version: @version,
-     elixir: "~> 1.3",
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     test_coverage: [tool: ExCoveralls],
-     preferred_cli_env: [coveralls: :test, "coveralls.travis": :test],
-     deps: deps(),
-     package: package(),
-     description: description(),
-     name: "ExPhoneNumber",
-     source_url: "https://github.com/socialpaymentsbv/ex_phone_number",
-     homepage_url: "https://github.com/socialpaymentsbv/ex_phone_number"]
+    [
+      app: :ex_phone_number,
+      version: @version,
+      elixir: "~> 1.6",
```

Comment:
Will it really stop working with Elixir below 1.6? Will it not work with `~> 1.3`?

Reply:
@szymon-jez It might work but the main reason for updating the version is that according to the doc [Compatibility and Deprecations](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Compatibility%20and%20Deprecations.md) the version 1.3 is no longer supported.

Also some dependencies like Jason require `~> 1.4`

Reply:
@josemrb So we at least have to go with `~> 1.4`, right? If so I suggest that version number after testing that it all works with it.